Authors.md

```diff
@@ -7,10 +7,9 @@ The Microsoft Mixed Reality Toolkit is a collaborative project containing contri
 - Addison Linville (radicalad)
 - ADP-JoNeff
 - afarchy
-- Against Lightning XR (againstlightning)
+- Against Lightning (AgainstLightning)
 - alandergrouse (alandergrouse)
 - Alex Floyd (elbuhofantasma)
-- Alexander Seeck (Alexees)
```

Comment:
I think this removal (and the SGHolospace one below) happens when a user removes the email address they used to commit from their GitHub account, so it's no longer linked. I wonder if we should try to maintain the entries?

Reply:
I can add re-add the names for now, but we can update the contributors script for next time

Reply:
I think we should maintain the names on the list